README.md

```diff
@@ -1,62 +1,105 @@
 # [👥 Yalies](https://yalies.io)
 
-> A website and API for getting information on students at Yale College.
+A website and API for getting information on students at Yale College.
 
 ![Screenshot](screenshot.png)
 
 ## Initial Setup
 
 To develop changes to the application, you'll need to run it locally for testing.
 
-This guide assumes, as prerequisites, that you have:
+This guide assumes, as prerequisites, that you have A MacOS or Linux-based OS (if you use Windows, you can still follow along, but some commands may be different).
 
-- A MacOS or Linux-based OS (if you use Windows, you can still follow along, but some commands may be different)
-- Access to, and basic understanding of, the terminal on your computer
-- Python 3
-  - Modern MacOS versions and most Linux distributions include Python 3 by default.
-  - If you use a version of MacOS with only Python 2, first [install Homebrew](https://brew.sh/#install), a helpful package manager for MacOS, then use it to install Python 3:
-  ```sh
-  brew install python3
-  ```
-- PostgreSQL, which is needed for working with our databases:
-  - If you're on MacOS, use Homebrew (or another package manager) to install PostgreSQL
-  ```sh
-  brew install postgresql
-  ```
+### Install Homebrew
+- If you're on a Mac, first [install Homebrew](https://brew.sh/#install).
+- If you're on Linux, follow along with your distro's package manager (`pacman`, `apt-get`, `yum`, etc).
 
-Next, make a "fork" of the `api` repository by clicking the Fork button in top right of repository page. This will make a copy of the codebase under your own personal user account that you'll be able to modify as you wish. When you make changes to your own repository, you can then send those changes in to be approved and merged into the main codebase. (Read on to learn how!)
+### Install Python
+This project requires Python version 3.10.7. You may already have Python on your machine—it comes preinstalled with macOS—but it may
```

Comment:
Did you have particular trouble using a different version? I Have 3.11.6 and I'm able to run it just fine. I really don't think we should necessarily force everyone to set up pyenv just for this purpose, but we can definitely recommend it as an option to try for those that are having trouble.

Reply:
Yeah, I had 3.11 on my machine by default, but I got an error. [Stackoverflow pointed me to using 3.10](https://stackoverflow.com/questions/74711405/importerror-cannot-import-name-getargspec-from-inspect-c-users-swapn-appd). Anyways, the docker container file says it uses 3.10, so we should probably match that on our local machines.

Reply:
Hm, okay. Still I would prefer that we recommend this as a solution if people encounter such an error rather than adding friction for everyone. What do you think?

Reply:
Also we don't actually use Docker. Someone started setting it up and never finished. I would love it if that could be finished though.

Reply:
I attempted to get Docker going and was unsuccessful - you can ignore the requirements there until that becomes operational again @ericyoondotcom